Jenkinsfile

```diff
@@ -30,7 +30,12 @@ node() {
         }
 
         stage('Building BlueOcean') {
-          sh "mvn clean install -B -DcleanNode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -Dmaven.test.failure.ignore -s settings.xml -Dmaven.artifact.threads=30"
+          try {
+            sh "mvn clean install -B -DcleanNode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -Dmaven.test.failure.ignore -s settings.xml -Dmaven.artifact.threads=30"
+          } catch(e) {
+            archive '*/target/code-coverage/**/*.html'
+            throw e;
+          }
```

Comment:
Would `post always` for the stage also work?

Reply:
This one isn't `declarative`, does post still work?

Reply:
... magic 8-ball says "no".

Reply:
yeah only for declarative